SWLOR.Game.Server/Service/ItemService.cs

```diff
@@ -305,10 +305,12 @@ public string OnModuleExamine(string existingDescription, NWPlayer examiner, NWO
         {
             BASE_ITEM_AMULET,
             BASE_ITEM_ARMOR,
+            BASE_ITEM_BRACER,
```

Comment:
I'm pretty sure changing this is going to cause problems elsewhere. Gloves are tricky because they're treated as weapons in some places, but armor in others. 

This change will make it so that AC and BAB mods can no longer be installed into any of these items. 

I think I'd recommend setting up a separate static hashset for the armor base items inside of the ModItem class.